sync-docs.js

```diff
@@ -38,7 +39,10 @@ const replaceMDImageUrl = (project, paths) => {
         .match(/\((.+?)\)/g)[0]
         .replace("(", "")
         .replace(")", "")
-        .replaceAll("../", "");
+        .replace("../", "")
+        .replace("../", "")
+        .replace("../", "")
+        .replace("../", "");
```

Comment:
I noticed this change is in https://github.com/apache/apisix-website/pull/205/commits/eda04d98b37a25dbd8dab5383b7e6372256c69fc to pass CI, but I can't get what's the meaning of this 🤣

Reply:
it's a trick here, to replace some paths like `../../../xxxx`, `../../xxxx`

Reply:
because replaceAll function is not available here, so I use a trick here. This JS file will get improved in the future.